### PR TITLE
NH-21449: Fix release action

### PR DESCRIPTION
**Pull request by xuan-cao-swi**
## Why make change?
This is for NH-21449 that to continue Maia's work.

## Impact
github action

### CODE REVIEW

File: .github/workflows/run_tests.yml [hunk 1]
```diff
@@ -90,10 +90,10 @@ jobs:
           - 5672:5672
         options: --health-cmd "rabbitmqctl node_health_check" --health-interval 10s --health-timeout 5s --health-retries 5
       mongo:
-        image: mongo:5
+        image: mongo:latest
         ports:
           - 27017:27017
-        options: "--health-cmd \"mongo --quiet --eval 'quit(db.runCommand({ ping: 1 }).ok ? 0 : 2)'\" --health-interval 10s --health-timeout 5s --health-retries 5"
+        options: --health-cmd "mongosh --quiet --eval 'quit(db.runCommand({ping:1}).ok ? 0:2)'" --health-interval 10s --health-timeout 5s --health-retries 5
```
Comment (cheempz) [inline .github/workflows/run_tests.yml:96]: Nice! I'm assuming this fixes the health check issue we saw with mongo:6.  Is there a test run to show it worked?

**Review by cheempz — COMMENTED** [review]
@xuan-cao-swi it looks like the only change here is the mongo:6 healthcheck, but has not addressed the review comment on the original branch: https://github.com/appoptics/appoptics-apm-ruby/pull/158

Also looks like you might be looking into a couple of issues as part of this ticket:

* redis failure -- could be instrumentation not covering newer client as you noted in Jira
* your commit message mentioned "maunal action on test on ubuntu" but I don't see changes about workflow trigger

Not opposed to covering a few different things in a single PR but let's have a discussion what you aim to cover in this PR and whether it's ready for review :)

File: .github/workflows/run_tests.yml (hunk 1 above)
Comment (xuan-cao-swi) [reply .github/workflows/run_tests.yml:96]: yes, I have tried with new mongo container.
on mongo 6, they use `mongosh` to enter the console.
and the `{ping:1}` works as same as `{ ping: 1 }`

**Review by cheempz — APPROVED** [review]
LGTM, thanks for the revisit @xuan-cao-swi!

**Pull request merged**